src/audio/AudioListener.js

```diff
@@ -20,6 +21,8 @@ function AudioListener() {
 
 	this.filter = null;
 
+	this.timeDelta = 0;
```

Comment:
Why is `timeDelta` public? Seems like it's only used inside `updateMatrixWorld()`?

Reply:
It's also used in `PositionalAudio. updateMatrixWorld()` via `this.listener.timeDelta`.

Reply:
I have not found another solution so far in order to pass in a time delta value to audio entities. At least the current implementation does not require a more tight coupling to the renderer.